docs/comparison/nbdb.md

```diff
@@ -22,28 +22,28 @@ kernelspec:
 Requirements:
 
 ```sh
-pip install scikit-learn sklearn-evaluation ploomber jupysql
+pip install scikit-learn sklearn-evaluation ploomber ploomber-engine jupysql nbformat jupytext
 ```
 
 ```{code-cell} ipython3
 from pathlib import Path
 
-# to train models in parallel
-from ploomber import DAG
-from ploomber.tasks import NotebookRunner
+import jupytext
+import nbformat
+
 from ploomber.products import File
-from ploomber.executors import Parallel, Serial
+from ploomber_engine import execute_notebook
+from ploomber_engine.tracking import track_execution
 
 # to produce parameter grid
 from sklearn.model_selection import ParameterGrid
 
 # to create SQLite database
 from sklearn_evaluation import NotebookDatabase
 ```
-
 ## Code
 
-`NotebookDatabase` indexes the output of tagged cells. In this example, we're using Python scripts (and tag cells using `# %% tags=["some-tag"]`), but the same concept applies for notebooks (`.ipynb`), [see here](https://docs.ploomber.io/en/latest/user-guide/faq_index.html#parameterizing-notebooks) to learn how to tag cells in `.ipynb` files.
+NotebookDatabase indexes the output of tagged cells. In this example, we're using Python scripts (and tag cells using # %% tags=["some-tag"]). We convert these scripts to notebooks using jupytext and nbformat, but the same concept applies for notebooks (.ipynb)— see here to learn how to tag cells in .ipynb files.
```

Comment:
Let's keep the "code" parts highlighted (with ``) so it will be easier to read.

Also, I think this line is a little bit confusing
```
see here to learn how to tag cells in .ipynb files.
```
Maybe we can make it shorter and as a title for the upcoming code?

Reply:
`see here to learn how to tag cells in .ipynb files.` : Please add the link of tutorial on tagging cells in jupyter

Reply:
`same concept` : which concept? I think these sentences can be broken into more concise and clear sentences, right now it's a bit confusing

Reply:
Will change now

Reply:
Okay, let me know if my most recent changes look better